src/odemis/driver/test/hamamatsurx_test.py

```diff
@@ -325,12 +325,12 @@ def test_MCPGain(self):
 
     def test_TimeRange(self):
         """Test time range VA for sweeping of streak unit."""
-        for timeRange in self.streakunit.timeRange.choices:
+        for timeRange in self.streakunit.timeRange.choices:  # values different from yaml due to floating point issues
             # change timeRange VA to value in range
-            self.streakunit.timeRange.value = util.find_closest(timeRange, self.streakunit.timeRange.choices)
+            self.streakunit.timeRange.value = timeRange
             self.assertAlmostEqual(self.streakunit.timeRange.value, timeRange)
 
-            tr2d = self.delaybox._metadata.get(model.MD_TIME_RANGE_TO_DELAY)
+            tr2d = self.delaybox._metadata.get(model.MD_TIME_RANGE_TO_DELAY)  # TODO MD not updated when HW VA changes?
```

Comment:
Yes, that's expected. It's up to the "client" to update the calibration.

Reply:
yes, forgot to remove this comment after we discussed